docs/core/linux-prerequisites.md

```diff
@@ -127,244 +155,417 @@ scl enable rh-dotnet20 bash
+    sudo apt-get update
+    ```
+
+    **Ubuntu 17.04**
+
+    ```bash
+    sudo sh -c 'echo "deb [arch=amd64] https://packages.microsoft.com/repos/microsoft-ubuntu-zesty-prod zesty main" > /etc/apt/sources.list.d/dotnetdev.list'
+    sudo apt-get update
+    ```
 
-## Install .NET Core for Ubuntu 14.04, 16.04, 16.10 & Linux Mint 17, 18 (64 bit)
+4. Install .NET Core.
 
-> [!Warning]
-> Before you start, remove any previous versions of .NET Core from your system.
+    ```bash
+    sudo apt-get install dotnet-sdk-2.0.0
+    ```
 
-### Add the dotnet apt-get feed
+4. Run the `dotnet --version` command to prove the installation succeeded.
 
-1. Set up the desired version host package feed.
+    ```bash
+    dotnet --version
+     ```
+
+# [.NET Core 1.x](#tab/netcore1x)
+
+2. Set up the desired version host package feed.
 
    **Ubuntu 14.04 / Linux Mint 17**
+
     ```bash
-    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list' 
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ trusty main" > /etc/apt/sources.list.d/dotnetdev.list'
     sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
     sudo apt-get update
     ```
+
     **Ubuntu 16.04 / Linux Mint 18**
+
     ```bash
-    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list' 
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list'
     sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
     sudo apt-get update
     ```
+
     **Ubuntu 16.10**
+
     ```bash
-    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ yakkety main" > /etc/apt/sources.list.d/dotnetdev.list' 
+    sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ yakkety main" > /etc/apt/sources.list.d/dotnetdev.list'
     sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys B02C46DF417A0893
     sudo apt-get update
     ```
-2. Install .NET Core.
-# [.NET Core 2.x](#tab/netcore2x)
 
-After host package feed setup, install .NET Core 2.0 on Ubuntu or Linux Mint:
-```bash
-sudo apt-get install dotnet-sdk-2.0.0
-```
-# [.NET Core 1.x](#tab/netcore1x)
+3. Install .NET Core 1.x on Ubuntu or Linux Mint:
+
+    ```bash
+    sudo apt-get install dotnet-dev-1.0.4
+    ```
+
+4. Run the `dotnet --version` command to prove the installation succeeded.
+
+    ```bash
+        dotnet --version
+    ```
+
+---
+
+ ## Install .NET Core for Debian 8 or Debian 9 (64 bit)
+
+To install .NET Core on Debian 8 or Debian 9 (64 bit):
+
+1. Remove any **previous preview** versions of .NET Core from your system.
```

Comment:
wondering if listing all steps inside the tab would look better

Reply:
Not sure.